frontend/src/components/commons/QuitTest.jsx

```diff
@@ -0,0 +1,168 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import { Button } from "react-bootstrap";
+import PopupBox, { BUTTON_TYPE, BUTTON_STATE } from "../commons/PopupBox";
+import SystemMessage, { MESSAGE_TYPE } from "../commons/SystemMessage";
+import LOCALIZE from "../../text_resources";
+import { deactivateTest } from "../../modules/TestStatusRedux";
+
+const styles = {
+  button: {
+    marginRight: 15
+  },
+  hr: {
+    width: "100%",
+    borderTop: "2px solid #96a8b2",
+    margin: "16px 0 16px 0"
+  },
+  checkboxZone: {
+    paddingTop: 8
+  },
+  startTestBtn: {
+    textAlign: "center",
+    marginTop: 32
+  }
+};
+
+const quitConditions = () => {
+  return [
+    { text: LOCALIZE.emibTest.testFooter.quitTestPopupBox.checkboxOne, checked: false },
+    { text: LOCALIZE.emibTest.testFooter.quitTestPopupBox.checkboxTwo, checked: false },
+    { text: LOCALIZE.emibTest.testFooter.quitTestPopupBox.checkboxThree, checked: false }
```

Comment:
Loading the LOCALIZE here causes the issue I noted in my other comment (i.e. that the language of the quit checkboxes is does not toggle language)

Reply:
replicated on master... resolving